src/html/classic/omp.xsl

```diff
@@ -9694,6 +9718,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
       data-sort-field="{/envelope/params/sort_field}"
       data-sort-stat="{/envelope/params/sort_stat}"
       data-sort-order="{/envelope/params/sort_order}"
```

Comment:
@timopollmeier shouldn't these old data-sort properties be removed?

Reply:
These are still used by some charts that only require sorting by one field.

Reply:
Ah overlooked the s at the end of the properties. Thanks for clarification!